src/Wrapper.js

```diff
@@ -216,6 +221,7 @@ export default (Component) => {
   };
 
   WrappedComponent.defaultProps = {
+    innerRef: null,
```

Comment:
Fixes the build which is currently broken here.

Same solution as is implemented in #124